docs/api/ShallowWrapper/dive.md

```diff
@@ -2,7 +2,8 @@
 
 Shallow render the one non-DOM child of the current wrapper, and return a wrapper around the result.
 
-NOTE: can only be called on wrapper of a single non-DOM component element node.
+NOTE: `dive` would throw an error if the wrapper contains more than one child, or if the (only one) child isn't a custom React component element node.
```

Comment:
Does "can only be called" not imply to you that it will throw an error?

"single" means "only one", and "non-DOM" means "custom React component".

Reply:
I agree, no need to change this sentence. I made a modification again.